main/vmaf/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "vmaf"
+pkgver = "3.0.0"
+pkgrel = 0
+build_style = "meson"
+meson_dir = "libvmaf"
+hostmakedepends = ["pkgconf", "meson", "nasm", "doxygen"]
```

Comment:
does `doxygen` generate any worthwhile documentation for end-users? `configure_args = ["-Denable_docs=false"]` at least seems to be an option and they probably aren't too valuable if also available online in the same form

also sort the list alphabetically